### PR TITLE
Fix uint64 bigint packing and unpacking on iOS

### PR DESCRIPTION
**Pull request by netoneko**
Follow up to #40

### CODE REVIEW

File: javascript/src/bigint.ts [hunk 1]
```diff
@@ -32,3 +32,27 @@ export function uint8ArrayToBigInt(v: Uint8Array): bigint {
     }
     return BigInt(hexStr);
 }
+
+export function uint64ToUint8Array(v: bigint): Uint8Array {
+    let hex = v.toString(16);
+    // if (16 < hex.length) {
```
Comment (noambergIL) [inline javascript/src/bigint.ts]: please uncomment. its good that if someone tries to translate a 65 bit (hex will be bigger than 16) number it fails rather then save only the first 64 bits.

File: javascript/src/data-view-wrapper.ts [hunk 2]
```diff
@@ -1,45 +1,42 @@
-import jDataView from "jdataview";
+// import jDataView, { Uint64 } from "jdataview";
```
Comment (noambergIL) [inline javascript/src/data-view-wrapper.ts]: remove comments.

**Comment by electricmonk** [comment]
@netoneko can you please add a comment above `IOSDataViewWrapper` explaining why it's needed?

**Comment by netoneko** [comment]
@electricmonk done

**Pull request merged**